runtime/queries/python/locals.scm

```diff
@@ -10,16 +10,20 @@
 
 ; Parameters
 (parameters
-  (identifier) @local.definition.variable.parameter)
+  (identifier) @local.definition.variable.parameter
+  (#not-match? @local.definition.variable.parameter "^(self|cls)$"))
 (parameters
   (typed_parameter
-    (identifier) @local.definition.variable.parameter))
+    (identifier) @local.definition.variable.parameter)
+    (#not-match? @local.definition.variable.parameter "^(self|cls)$"))
```

Comment:
do we need to put those no match queries everywhere ?

I think I have been able to fix the issue by just adding the below at row 29 

```scheme
(parameters
  (identifier) @local.definition.variable.builtin
  (#match? @local.definition.variable.builtin "^(self|cls)$"))
```